File: Cpp/119_Pascal's_Triangle_II.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> result(rowIndex+1);
        result[0] = 1;
        result[rowIndex] = 1;
        for(int i=0; i<=rowIndex; ++i){
            result[i] = 1;
            for(int j=i-1; j>0; --j){
                result[j] = result[j-1] + result[j];
            }
        }
        
        return result;
    }
};

File: Kotlin/119_Pascal's_Triangle_II.kt
class Solution {
    fun getRow(rowIndex: Int): List<Int> {
        val result = MutableList(1){1}
        
        for(i in 1..rowIndex){
            result.add(1)
            for(j in i - 1 downTo 1){
                result.set(j, result[j] + result[j - 1])
            }
        }
        
        return result
    }
}